sympy/parsing/c/c_parser.py

```diff
@@ -784,6 +838,8 @@ def transform_binary_operator(self, node):
                         # '(' is found
                         while (len(operators_stack) != 0
                             and operators_stack[-1] != '('):
+                            if len(combined_variables_stack) < 2:
+                                raise NotImplementedError()
```

Comment:
Can you add an appropriate message to all the `NotImplementedError` here?

Reply:
Does that look good now? I couldn't find a better error message! :sweat_smile:

Reply:
It's alright, but it doesn't really tell the user what part of the expression is not supported. 
I think most of these are raised for unary operators, there you can just mention unary operators as a part of binary operations are not supported yet.
And I think another one was for unsupported token types, there you can just mention that the token type is not supported yet.

Reply:
Alright!